bin/bundle

```diff
@@ -8,107 +8,103 @@
 # this file is here to facilitate running it.
```

Comment:
There shouldn't really be any need to rubocop system files as these are set by rails and a should be updated with any major upgrade. It is just overhead.

Reply:
I am not suggesting reverting at this stage but the bin folder should be ignored.